src/Pages/Form/Form.tsx

```diff
@@ -0,0 +1,168 @@
+        </div>
+        <div className='input-container'>
+          <label htmlFor='manufacturer-address'>Manufacturer address :</label>
+          <input type='text' className='manufacturer-info' id="manufacturer-address" value={form.Manufacturer_address}></input>
+        </div>
+        <div className='input-container'>
+          <label htmlFor='manufacturer-website'>Manufacturer website :</label>
+          <input type='text' className='manufacturer-info' id="manufacturer-website" value={form.Manufacturer_website}></input>
+        </div>
+        <div className='input-container'>
+          <label htmlFor='manufacturer-phone'>Manufacturer phone number :</label>
+          <input type='text' className='manufacturer-info' id="manufacturer-phone" value={form.Manufacturer_phone_number}></input>
+        </div>
+      </div>
+      {/* ----------     Fertiliser section     ---------- */}
+      <div className='Fertiliser-container data-section'>
+        <h1 className='title underlined'>Fertiliser information</h1>
+        <div className='input-container'>
+          <label htmlFor='fertiliser-name'>Fertiliser name :</label>
+          <input type='text' className='fertiliser-info' id="fertiliser-name" value={form.Fertiliser_name}></input>
+        </div>
+        <div className='input-container'>
+          <label htmlFor='fertiliser-registr_num'>Fertiliser registration number :</label>
+          <input type='text' className='fertiliser-info' id="fertiliser-registr_num" value={form.Fertiliser_registration_number}></input>
+        </div>
+        <div className='input-container'>
+          <label htmlFor='fertiliser-lot_number'>Fertiliser lot number :</label>
+          <input type='text' className='fertiliser-info' id="fertiliser-lot_number" value={form.Fertiliser_lot_number}></input>
+        </div>
+        <div className='input-container'>
+          <label htmlFor='fertiliser-NPK'>Fertiliser NPK :</label>
+          <input type='text' className='fertiliser-info' id="fertiliser-NPK" value={form.Fertiliser_NPK}></input>
+        </div>
+        <div className='input-container'>
+          <label htmlFor='fertiliser-precau_FR'>Fertiliser precautionary FR :</label>
+          {/**
+          <div className="textbox-container">
+            <textarea
+              value={text}
+              onChange={handleTextChange}
+              className='text-box'
+            />
+            {isTextTooLong && (
+              <img
+                src={openIcon}
+                alt="Ouvrir l'overlay"
+                className="open-icon"
+                onClick={() => setShowOverlay(true)}
+              />
+            )}
+
+            {showOverlay && (
+              <Modal text={text} handleTextChange={handleOverlayTextChange} close={handleOverlayClick} />
+            )}
+          </div>
+           */}
+        </div>
+      </div>
+
+
+      
+    </div>
+    
+  );
+};
```

Comment:
Not sure if this is the more appropriate way to do it. The best solution would be to put a for each that generate value and section automatically with the reading of the json file! Going to make an issue for later.